File: sdkwork-backend-sdk-go/types/plus_user_address_form.go
package types

// Address creation form data
type PlusUserAddressForm struct {
	Name string `json:"name"`
	Phone string `json:"phone"`
	CountryCode string `json:"countryCode"`
	ProvinceCode string `json:"provinceCode"`
	CityCode string `json:"cityCode"`
	DistrictCode string `json:"districtCode"`
	AddressDetail string `json:"addressDetail"`
	PostalCode string `json:"postalCode"`
	IsDefault bool `json:"isDefault"`
	Tags TagsContent `json:"tags"`
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_user_address_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusUserAddressForm:
    """Address creation form data"""
    name: str
    phone: str
    country_code: str = None
    province_code: str = None
    city_code: str = None
    district_code: str = None
    address_detail: str
    postal_code: str = None
    is_default: bool = None
    tags: TagsContent = None
